packages/expansion-hub/src/open-rev-hub.ts

```diff
@@ -37,8 +38,12 @@ export async function openParentExpansionHub(serialNumber: string, moduleAddress
         moduleAddress = addresses.parentAddress;
     }
 
-    await parentHub.open(moduleAddress);
-    await parentHub.queryInterface("DEKA");
+    try {
+        await parentHub.open(moduleAddress);
+        await parentHub.queryInterface("DEKA");
+    } catch(e: any) {
+        if(e.errorCode == -2) throw new NoExpansionHubWithAddressError(moduleAddress);
```

Comment:
Please document what this error code is in a comment here

Reply:
Documented